### PR TITLE
Feature/edit profile basics

### PR DESCRIPTION
**Pull request by dupreesi**

### CODE REVIEW

File: server/__test__/api/updateUserInfo.js [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+const app = require("../../app");
+
+// ROUTES
+const {
+  API_LOGIN_URL,
+  API_UPDATE_USERINFO_URL,
+} = require("../../../client/src/constants/apiRoutes");
+
+describe("Testing for post future message route", () => {
```
Comment (Oliversw) [inline server/__test__/api/updateUserInfo.js]: Might be worth changing this a little 😉

**Comment by dupreesi** [comment]
@MohammedYehia  @Oliversw I think that @othman-shamla is taking over the branch so should be resolving open issues.. Thanks for checking the PR!

**Comment by MohammedYehia** [comment]
@teenie-quaggard Please merge if everything is fine

**Comment by teenie-quaggard** [comment]
Hi @MohammedYehia !

Super nice work here! Sorry it took me a while to go through everything...

A couple things to note, but will make new issues for:
1. Buttons for the image upload / delete are not focus-able (by tab)
2. Need to create some sort of client side error if they try to change their email to something that we already have stored in the database. It causes this error
Screen Shot 2019-08-29 at 11 33 17
3. Also have been getting these errors in my console. Do you get them as well?
Screen Shot 2019-08-29 at 11 43 27

**Comment by MohammedYehia** [comment]
@teenie-quaggard Yes we should handle the error and do some friendly error message even though I am not getting these error 😕 
PS: try rebuilding the database and try again with the email issue

as for the last one, no error for me in the browser console

**Comment by teenie-quaggard** [comment]
Ok, let's merge this! I've opened up the other issues ☺️

**Pull request merged**